### ISSUE TITLE
aws-ram: Broken link in documentation

### ISSUE THREAD

**Issue by ryanolee** [aws/aws-cdk#24273]
### Describe the issue

# What 
There is a broken link labelled "Sharing with IAM roles and users" linking to "https://ram/latest/userguide/permissions.html#permissions-rbp-supported-resource-types" on the ram documentation page https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ram.CfnResourceShare.html.

# Other considerations
Considering this seems like this could be a more common issue considering a very similar issue was found with another CFN sourced link was opened earlier [here](https://github.com/aws/aws-cdk/issues/24054) it might be worth re raising this a wider ticket to check all the documentation derived from the CF docs to check for and then fix dead links due to double slashes (//) being in the URL or other reasons :thinking: 

### Links

https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ram.CfnResourceShare.html

**Comment by peterwoodworth** [comment]
Yes, let's track this as a wider issue. If anyone comes across broken links in l1 docs in the future, let's track that here
